mvp/src/main/java/com/hannesdorfmann/mosby3/mvp/delegate/ViewGroupDelegateCallback.java

```diff
@@ -50,4 +50,10 @@
    * @since 3.0
    */
   Context getContext();
+
+  /**
+   * A android.view.View can added (in)directly to a Activity or a WindowManager,
+   * in which case only destroy Presenter is needed, not release
+   */
+  boolean isViewOnActivity();
```

Comment:
How is this intended to work? It should return false if used without activity?

Reply:
Thanks. I will make it more explicit